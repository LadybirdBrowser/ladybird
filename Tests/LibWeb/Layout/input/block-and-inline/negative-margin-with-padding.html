<!DOCTYPE html>
<style>
body {
    border: 10px solid red;
}
#a {
    padding-bottom: 150px;
}
#b {
    font-size: 5em;
    margin: -240px 0 0;
}
</style>
<div id="a"></div>
<div id="b">foo</div>
bar
