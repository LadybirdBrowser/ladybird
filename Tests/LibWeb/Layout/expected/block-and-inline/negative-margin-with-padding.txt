Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x56 [BFC] children: not-inline
    BlockContainer <body> at (18,18) content-size 764x20 children: not-inline
      BlockContainer <div#a> at (18,18) content-size 764x0 children: not-inline
      BlockContainer <(anonymous)> at (18,168) content-size 764x0 children: inline
        TextNode <#text>
      BlockContainer <div#b> at (18,-72) content-size 764x92 children: inline
        frag 0 from TextNode start: 0, length: 3, rect: [18,-72 135.78125x92] baseline: 69.984375
            "foo"
        TextNode <#text>
      BlockContainer <(anonymous)> at (18,20) content-size 764x18 children: inline
        frag 0 from TextNode start: 1, length: 3, rect: [18,20 27.640625x18] baseline: 13.796875
            "bar"
        TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600] overflow: [0,-72 800x672]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x56] overflow: [0,-72 800x240]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x40] overflow: [18,-72 764x240]
      PaintableWithLines (BlockContainer<DIV>#a) [18,18 764x150]
      PaintableWithLines (BlockContainer(anonymous)) [18,168 764x0]
      PaintableWithLines (BlockContainer<DIV>#b) [18,-72 764x92]
        TextPaintable (TextNode<#text>)
      PaintableWithLines (BlockContainer(anonymous)) [18,20 764x18]
        TextPaintable (TextNode<#text>)

SC for Viewport<#document> [0,0 800x600] [children: 1] (z-index: auto)
 SC for BlockContainer<HTML> [0,0 800x56] [children: 0] (z-index: auto)
